src/act.other.cpp

```diff
@@ -1408,7 +1408,10 @@ ACMD(do_toggle)
         if (ch->desc->pProtocol)
           ch->desc->pProtocol->do_coerce_ansi_capable_colors_to_ansi = FALSE;
       }
-    } else {
+    } else if (is_abbrev(argument, "RP") || is_abbrev(argument, "RPING") || is_abbrev(argument, "roleplay") || is_abbrev(argument, "visible")) {
```

Comment:
Using 'rp' to toggle this won't actually work (it's shadowed by the RPE flag). Let's drop most of these and call it 'nosocialization' so it's clearer what its function is. ("nosocialization", "socialization")

You also need to add messages to the end of the big block above, as they're what's printed when this command is run. The game will have undefined behavior (will likely crash) if run in its current state.